frontend/javascripts/admin/organization/pricing_plan_utils.ts

```diff
@@ -29,7 +29,7 @@ export const powerPlanFeatures = [
 export const maxInludedUsersInBasicPlan = 3;
 
 export function getActiveUserCount(users: APIUser[]): number {
-  return users.filter((user) => user.isActive && !user.isSuperUser).length;
+  return users.filter((user) => user.isActive && !user.isSuperUser && !user.isGuest).length;
```

Comment:
I’m not sure that superUsers should be skipped here. I think I’d rather check the isUnlisted property here for less confusion.
E.g. I was surprised to see that the sample user doesn’t count in their own sample_organization.

While this was not introduced in this PR, I think it makes sense to change that now.

I think that’s also what we use in the backend to determine if the limit is reached

Reply:
Ok I now exposed the `isUnlisted` field to the frontend. IMO this should be ok as this properly is nothing sensitive that must be protected